swabbie-web/src/main/kotlin/com/netflix/spinnaker/swabbie/controllers/ResourceController.kt

```diff
@@ -196,4 +196,22 @@ class ResourceController(
         && workConfiguration.location == namespace.region
     } ?: throw NotFoundException("No configuration found for $namespace")
   }
+
+  /**
+   * Recalculate deletion timestamp to @retentionS seconds in the future for the
+   * oldest @numResources that are marked
+   */
+  @RequestMapping(value = ["/recalculate/{namespace}/"], method = [RequestMethod.PUT])
+  fun recalculate(
+      @PathVariable namespace: String,
+      @RequestParam(required = true) retentionS: Long,
+      @RequestParam(required = false, defaultValue = "100") numResources: Int
```

Comment:
Is it safer to just not have a default value for `numResources`? Doesn't seem like a stretch to ask operators to be explicit about what they're like done?

Reply:
Sure, that makes sense! Changing.